src/settings/settings.html

```diff
@@ -9,6 +9,17 @@
 </head>
 
 <body>
+    <main class="main">
+        <section class="section">
+            <h1 class="section__header">Settings</h1>
+            <div class="section__input">
+                <label class="section__label">Your name:</label>
+                <input class="section__input--playerName" type="text" placeholder="Player" />
```

Comment:
Wouldn't it be easier with `input`?

Reply:
Of course, I'm aware of the fact I said earlier `section__input` is enough so it is only a suggestion. You don't need to change it this time if you don't want to.